lib/pthreads/examples/jointask_example.nit

```diff
@@ -0,0 +1,47 @@
+# This file is part of NIT (http://www.nitlanguage.org).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	 http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Simple example using threadpool
+module jointask_example
+
+import threadpool
+
+# Task computing a string
+class IntTask
+	super JoinTask
+
+	# Sleeping time
+	var sec: Int
+	# result of `self` execution
+	var value: Int
+	redef fun main do
+		nanosleep(sec, 0)
+		value += 10
+	end
+end
+
+var tp = new ThreadPool
+var t0 = new IntTask(10, 55)
+var tasks = new Array[IntTask]
+for i in 5.times do
+	tasks.add(new IntTask(1, 0))
+end
+tp.execute(t0)
+for t in tasks do
+	tp.execute(t)
+	t.join
+	print t.value
+end
+t0.join
+print t0.value
```

Comment:
Next time, please use more expressive `values`. Here it could have been a String instead of an Int, and the values something like `"First, long task", "Small task 1", etc.`